project/frontend/src/components/PreferenceForm.js

```diff
@@ -52,7 +52,10 @@ async function getLocalCuisines(currLocation) {
  */
 function PreferenceForm(props) {
   const { currLocation, locationName } = props;
-
+  const currHour = parseInt(
+    new Date(Date.now()).toTimeString().substring(0, 2)
+  );
+  const isOpenCheckedDefault = currHour >= 10 && currHour <= 22;
```

Comment:
probably worth leaving s comment here with just "open now is checked by default if the current time is between 10am and 10pm" -- the >= 10 && <= 22 check might not give that info at just first glance